docker/README.md

```diff
@@ -3,46 +3,86 @@
 
 If you want to know more about images and the building process you can find it in [Docker's documentation](https://docs.docker.com/userguide/dockerimages/).
     
-## Run the container
+### Run the container
+
+The following line will run the container exposing port `1026`, give it a name -in this case `orion1`, link it to a mongodb docker, and present a bash prompt. This uses the image built in the previous section.
 
-The following line will run the container exposing port `1026`, give it a name -in this case `orion1`, link it to a mongodb docker, and present a bash prompt.
+	  sudo docker run -d --name orion1 --link mongodb:mongodb -p 1026:1026 orion -dbhost mongodb
 
-	  docker run -d --name orion1 --link mongodb:mongodb -p 1026:1026 orion -dbhost mongodb
+If did not build the image yourself and want to use the one on Docker Hub use the following command:
+
+	  sudo docker run -d --name orion1 --link mongodb:mongodb -p 1026:1026 fiware/orion -dbhost mongodb
+
+> **Note**
+> Keep in mind that if you use this last command you get access to the tags and specific versions of Orion. For example, you may add `fiware/orion:0.22` if you need that particular version. If you don't specify a version you are pulling from `latest`.
 
 As a result of this command, there is a context broker listening on port 1026 on localhost. Try to see if it works now with
 
 	curl localhost:1026/version
 
+
+
 A few points to consider:
 
 * The name `orion1` can be anything and doesn't have to be related to the name given to the docker image in the previous section.
-* `--link mongodb:mongodb` assumes there is a docker container running a MongoDB image in your system, whose name is `mongodb`. In case you need one type `docker run --name mongodb -d mongo:2.6`.
+* `--link mongodb:mongodb` assumes there is a docker container running a MongoDB image in your system, whose name is `mongodb`. In case you need one type `sudo docker run --name mongodb -d mongo:2.6`.
 * In `-p 1026:1026` the first value represents the port to listen in on localhost. If you wanted to run a second context broker on your machine you should change this value to something else, for example `-p 1027:1026`.
-* Anything after the name of the container image (in this case `orion`) is interpreted as a parameter for the Orion Context Broker. In this case we are telling the broker where the MongoDB host is, represented by the name of our other MongoDB container. Take a look at the [documentation](../.../.../doc/admin/cli.md) for other command-line options.
+* Anything after the name of the container image (in this case `orion`) is interpreted as a parameter for the Orion Context Broker. In this case we are telling the broker where the MongoDB host is, represented by the name of our other MongoDB container. Take a look at the [documentation](../doc/manuals/admin/cli.md) for other command-line options.
 
```

Comment:
we still have a local link which will not be working inside docker hub, please check